crates/turborepo-telemetry/src/events/command.rs

```diff
@@ -128,9 +137,9 @@ impl CommandEventBuilder {
     }
 
     // prune
-    pub fn track_prune_method(&self, is_docker: bool) -> &Self {
+    pub fn track_prune_arg_method(&self, is_docker: bool) -> &Self {
```

Comment:
I'm a little confused by the general name of this method. When it seems like it's `docker` specific.

I do feel like we could use `track_arg_usage("docker", docker)` in `prune.rs` to achieve the same thing and unify the values of `arg:*` a bit to either `"set"` or `"default"`

Reply:
Good call here, I started out with really specific events for arguments to keep things easily traceable, but as I've been adding more and more I'm deferring back to general, now I just have a `track_arg_usage` (in this PR), as well as `track_arg_value` (in a future PR). 

Refactored this to use the general approach.